README.md

```diff
@@ -0,0 +1,144 @@
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📖 [Bookstore]
+
+
+**[Bookstore]**  is an application that allows a user to:
+
+    Display a list of books
+    Add a book.
+    Remove a selected book.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
```

Comment:
- [ ] Also, please consider adding the table of contents from the original README template, it is important to have it as it keeps all of your README sections easily accessible by other people browsing your documentation! 🤓 🔧